documentation/proc-pages/physics-models/fusion_reactions/plasma_bosch_hale.md

```diff
@@ -0,0 +1,92 @@
+# Bosch-Hale Methods
+
+These methods are still kept in `physics_functions.py` but outside the [`FusionReactionRate`](plasma_reactions.md) class.
+
+## Bosch-Hale Constants | `BoschHaleConstants`
+
+The `BoschHaleConstants` class is a data structure designed to hold the constants required for the Bosch-Hale calculation for a given fusion reaction.
+
+### Attributes
+
+- **bg (float)**: Represents the Gamow energy parameter.
+- **mrc2 (float)**: Represents the reduced mass energy term.
+- **cc1 (float)**: Coefficient for the first term in the Bosch-Hale polynomial.
+- **cc2 (float)**: Coefficient for the second term in the Bosch-Hale polynomial.
+- **cc3 (float)**: Coefficient for the third term in the Bosch-Hale polynomial.
+- **cc4 (float)**: Coefficient for the fourth term in the Bosch-Hale polynomial.
+- **cc5 (float)**: Coefficient for the fifth term in the Bosch-Hale polynomial.
+- **cc6 (float)**: Coefficient for the sixth term in the Bosch-Hale polynomial.
+- **cc7 (float)**: Coefficient for the seventh term in the Bosch-Hale polynomial.
+
+--------------------------
+
+## Volumetric Fusion Rate | `bosch_hale_reactivity()`
+
+This function calcualtes the relative velocity fusion reactivity $\langle \sigma v \rangle$ for each point in the plasma profile based on the temperature.
+
+ |  Input Variable             |    |
```

Comment:
add "Variable Name" to second table heading?

Reply:
Added